web/models/employee.ex

```diff
@@ -61,4 +68,11 @@ defmodule Appointments.Employee do
     |> DB.add_reset_token(key)
   end
 
+  @doc """
+  Returns the full name of the employee
+  """
+  def full_name(employee) do
+    "#{employee.first_name} #{employee.last_name}"
```

Comment:
I was hoping you knew how, I was just guessing🤓

Sent from my iPhone




On Fri, May 20, 2016 at 3:33 PM -0700, "Hrafn Eiríksson" <notifications@github.com> wrote:












In web/models/employee.ex:
> @@ -61,4 +68,11 @@ defmodule Appointments.Employee do
>      |> DB.add_reset_token(key)
>    end
>  
> +  @doc """
> +  Returns the full name of the employee
> +  """
> +  def full_name(employee) do
> +    "#{employee.first_name} #{employee.last_name}"



Eeeehm... If it can you're gonna have to explain how to do that to me! 😖 



—
You are receiving this because you commented.
Reply to this email directly or view it on GitHub